src/main/java/com/solr/clientwrapper/domain/service/OrderPojoSolrService.java

```diff
@@ -0,0 +1,39 @@
+package com.solr.clientwrapper.domain.service;
+
+import java.util.List;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import com.solr.clientwrapper.domain.port.api.SolrOrderServicePort;
+import com.solr.clientwrapper.domain.port.spi.SolrOrderPersistencePort;
+import com.solr.clientwrapper.infrastructure.entity.SolrOrderPojo;
+
+@Service
+@Transactional
+public class OrderPojoSolrService implements SolrOrderServicePort {
+
+	private final SolrOrderPersistencePort solrOrderPersistencePort;
+
+	public OrderPojoSolrService(SolrOrderPersistencePort solrOrderPersistencePort) {
+		this.solrOrderPersistencePort = solrOrderPersistencePort;
+	}
+
+	@Override
+	public Page<SolrOrderPojo> findByOrderDescription(String searchTerm, Pageable pageable) {
+		return solrOrderPersistencePort.findByOrderDescription(searchTerm, pageable);
```

Comment:
rename the methods properly(give meaningful names that will explain the purpose)

Reply:
Done.